pkg/mdformatter/mdformatter_test.go

```diff
@@ -147,3 +147,18 @@ func TestCheck_SoftWraps(t *testing.T) {
 	testutil.Ok(t, err)
 	testutil.Equals(t, string(exp), diff.String())
 }
+
+func TestFormat_NoGofmt(t *testing.T) {
```

Comment:
Thanks for the separate test case. I would also check what happens to this go code sample when --no-gofmt is disabled, just to ensure we aren't changing default behavior